app/lib/frontend/templates/package_misc.dart

```diff
@@ -207,14 +208,20 @@ String renderLabeledScores(PackageView view, {String version}) {
   });
 }
 
-/// Renders the `views/pkg/labeled_score.mustache` template.
 String _renderLabeledScore(String label, int value, String sign) {
-  return templateCache.renderTemplate('pkg/labeled_score', {
-    'label': label,
-    'has_value': value != null,
-    'value': value ?? '--',
-    'sign': sign,
-  });
+  return dom.fragment([
+    dom.div(
+      classes: ['packages-score-value', if (value != null) '-has-value'],
+      children: [
+        dom.span(
+            classes: ['packages-score-value-number'],
+            children: [dom.text(value?.toString() ?? '--')]),
+        dom.span(
+            classes: ['packages-score-value-sign'], children: [dom.text(sign)]),
```

Comment:
nit, but I would use tailing comma in these cases... Not that it's important :D 
```suggestion
        dom.span(
          classes: ['packages-score-value-number'],
          children: [dom.text(value?.toString() ?? '--')],
        ),
        dom.span(
          classes: ['packages-score-value-sign'],
          children: [dom.text(sign)],
        ),
```